src/components/character-details/CharacterDetails.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { getOneCharacter } from '../../services/hey-arnold-api';
+import PropTypes from 'prop-types';
+
+export default class CharacterDetails extends Component {
+  static propTypes = {
+    match: PropTypes.shape({
+      params: PropTypes.shape({
+        id: PropTypes.string.isRequired
+      })
+    })
+  };
+  
+  state = {
+    character: ''
+  }
+
+  componentDidMount = async() => {
+    const characterId = this.props.match.params.id;
+    await getOneCharacter(characterId)
```

Comment:
You don't wanna mix aysnc/await with .then - choose one or the other

Reply:
Is "this." ok? I'm not seeing where I have .then.

Reply:
Yep that's good!